disperser/dataapi/v2/server_v2.go

```diff
@@ -210,10 +210,34 @@ func (s *ServerV2) Start() error {
 	}
 
 	router := gin.New()
+
+	// Add recovery middleware (best practice according to Cursor)
+	router.Use(gin.Recovery())
+
 	basePath := "/api/v2"
 	docsv2.SwaggerInfoV2.BasePath = basePath
 	docsv2.SwaggerInfoV2.Host = os.Getenv("SWAGGER_HOST")
 
+	// Configure CORS
```

Comment:
The order to set this up matters?

Reply:
See :233 CORS middleware needs to be created before routes.